modules/ROOT/pages/patterns/reference.adoc

```diff
@@ -760,6 +760,11 @@ upperBound ::= unsignedInteger
 unsignedInteger ::= [0-9]+
 ----
 
+[NOTE]
+====
+The `unsignedInteger` must not be larger than the Java constant `Long.MAX_VALUE`.
```

Comment:
Maybe this is not necessary, but helpful if readers are both unaware and interested in what the Long.MAZ_VALUE is? If you agree - same comment goes for below note. 

```suggestion
The `unsignedInteger` must not be larger than the Java constant `Long.MAX_VALUE` (2^63-1).
```

Reply:
Good idea, added.